apps/api/src/domain/models/media-item/commands/create-media-item/create-media-item.command.ts

```diff
@@ -102,4 +102,22 @@ export class CreateMediaItem implements ICommandBase {
     })
     // this only is specified when the MIMEType is for an `Audio Item` or `Video`.
     readonly lengthMilliseconds?: number;
+
+    @NonNegativeFiniteNumber({
+        label: 'height (px)',
+        description: 'the height of the media item in pixels',
+        isOptional: true,
+    })
+    // this only is specified with the MIMEType is for a `Photograph`
+    // what about video?
```

Comment:
We can add this later when we need it.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186725983) is a story for that.